spec/system/purchase_system_spec.rb

```diff
@@ -216,6 +216,13 @@
           visit edit_purchase_path(@user.organization.short_name, purchase)
           expect(page).to have_content("Still haven't found what you're looking for")
         end
+
+        it "The storage location is correct" do
+          storage1 = create(:storage_location, name: "storage1")
+          purchase = create(:purchase, storage_location: storage1)
+          visit edit_purchase_path(@organization.to_param, purchase)
+          expect(page).to have_content("storage1")
```

Comment:
I'm pretty sure this test will pass regardless of which location is selected. To verify, try taking your code in the previous files out and run the test - if it's still green, the test isn't useful. You'd need to find a better way to indicate that the right storage location is selected (perhaps via an `option selected` matcher.

Reply:
Yes you were correct, @dorner. I've updated the test to make sure the expected storage is selected. Thanks for all the feedback, let me know if there's anything else that can be improved on.